### DISCUSSION TITLE
简体中文行高问题自己修改解决

### DISCUSSION THREAD

**Discussion by hiforrest** [IBM/plex#630] [General]
官方修复看起来遥遥无期（[相关反馈](https://github.com/IBM/plex/issues/614)已经发布一个月了，还没有动静），不想等，于是自己动手用 FontForge 改了下数值，效果不错！目前只改了常规和 Medium 两个字重（够用就行）。

需要的小伙伴可以直接用我分享的文件，想自己动手的也可以照着改，都没问题！

![用fontforge反馈修改ibm plex sans sc行高问题](https://github.com/user-attachments/assets/6530241b-6d2f-4ce1-9f68-9bb95112ce2b)

修改后的字体：
https://www.123865.com/s/jsJPjv-FNkMd

补充说明：

- 并非专业字体设计师，修改后的字体可能有其他未知的问题
- 仅自己在文本编辑器中测试

**Comment by devkamiki** [comment]
非常感谢！

**Comment by shirok1** [comment]
https://github.com/shirok1/nogaplex

我实现了用 Python 的 fonttools 包直接 patch otf 应用了你的参数，这样就不需要安装 FontForge 了

下面是 Emacs 使用前后的效果图

<img width="397" height="283" alt="image" src="https://github.com/user-attachments/assets/aa492852-e809-4fee-ab34-a8e413a642b6" />

<img width="406" height="256" alt="image" src="https://github.com/user-attachments/assets/ecda472c-d50a-4873-9e21-be1b598833df" />
